migrations/manual/educandu-2021-12-13-01-make-cdn-urls-generic.js

```diff
@@ -0,0 +1,55 @@
+async function updateAll(collection, query, updateFn) {
```

Comment:
It's funny to see how we deleted this code from the migrations/helpers, and now we keep copying it :)).

Reply:
Yeah, I thought the same